### PR TITLE
feat(multiplayerpiano.net): add presence

### PR DESCRIPTION
**Pull request by AntoKek**
## Description 
Added a presence for [multiplayerpiano.net](multiplayerpiano.net), a website used for playing piano in custom lobbies with other people. \
There are settings for both showing the name of the room and for showing a button that allows other users to join the room.\
The presence keeps track of the user's status on the website in order to determine whether they're active or AFK.

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

### AFK, Both settings disabled

</details>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.MD
Comment (theusaf) [inline .github/PULL_REQUEST_TEMPLATE.MD:1]: Ensure to revert changes to this file

File: websites/M/MultiplayerPiano.net/metadata.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		"id": "224864947480297472",
+		"name": "antokek"
+	},
+	"service": "MultiplayerPiano.net",
+	"description": {
+		"en": "An online website used for playing piano with friends. It supports input from both MIDI keyboards and QWERTY keyboards."
+	},
+	"url": "multiplayerpiano.net",
+	"version": "1.0.0",
+	"logo": "https://cdn.discordapp.com/icons/813217128529264710/911b06d8a1a9af24fac16b843165cc0f.png",
```
Comment (theusaf) [inline websites/M/MultiplayerPiano.net/metadata.json:13]: The [logo](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) should be a 512x512px image

File: websites/M/MultiplayerPiano.net/presence.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		clientId: "1240257888514080830",
+	}),
+	strings = presence.getStrings({
+		play: "presence.playback.playing",
+		pause: "presence.playback.paused",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets { // Other default assets can be found at index.d.ts
+	Logo = "logo",
```
Comment (theusaf) [inline websites/M/MultiplayerPiano.net/presence.ts:11]: Use image links over Discord asset names

File: websites/M/MultiplayerPiano.net/presence.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    const roomNameSetting = await presence.getSetting<boolean>("showRoomName");
+    return roomNameSetting;
+}
+async function getShowStatus(): Promise<boolean> {
+    const roomNameSetting = await presence.getSetting<boolean>("showStatus");
+    return roomNameSetting;
+}
+
+function getRoomName(): string {
+	const url = window.location.href;
```
Comment (theusaf) [inline websites/M/MultiplayerPiano.net/presence.ts:28]: Use `document.location` instead of `window.location`

**Review by theusaf — CHANGES_REQUESTED** [review]
Make sure to resolve automated checks

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 4]
```diff
@@ -16,4 +16,4 @@
 
 
 
-</details>
+</details>
```
Comment (DarkVillager) [inline .github/PULL_REQUEST_TEMPLATE.MD:19]: huh

File: websites/M/MultiplayerPiano.net/metadata.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "224864947480297472",
+		"name": "antokek"
+	},
+	"service": "MultiplayerPiano.net",
+	"description": {
+		"en": "An online website used for playing piano with friends. It supports input from both MIDI keyboards and QWERTY keyboards."
```
Comment (DarkVillager) [inline websites/M/MultiplayerPiano.net/metadata.json:9]: Official description.
```suggestion
		"en": "An online piano you can play alone or with others in real-time. MIDI support, 88 keys, velocity sensitive. You can show off your skill or chat while listening to others play."
```

File: websites/M/MultiplayerPiano.net/presence.ts [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+	else presenceData.details = "Playing piano";
+
+	if (await getShowRoomName())
+		presenceData.state = `in room "${getRoomName()}"`;
+
+	if (await getShowJoinButton()) {
+		presenceData.buttons = [
+			{
+				label: "Join Room",
+				url: `https://multiplayerpiano.net/?c=${getRoomName()}`,
```
Comment (DarkVillager) [inline websites/M/MultiplayerPiano.net/presence.ts:46]:
```suggestion
				url: href,
```

File: websites/M/MultiplayerPiano.net/presence.ts [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+function getAFK(): boolean {
+	return document.querySelector('.name.me [id^="afktag-"]') !== null;
+}
+
+const presenceData: PresenceData = {
+	largeImageKey: Assets.Logo,
+	startTimestamp: Math.floor(Date.now() / 1000),
+};
+
+presence.on("UpdateData", async () => {
```
Comment (DarkVillager) [inline websites/M/MultiplayerPiano.net/presence.ts:35]:
```suggestion
presence.on("UpdateData", async () => {
cons {href} = document.location;
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/M/MultiplayerPiano.net/presence.ts [hunk 8]
```diff
@@ -0,0 +1,54 @@
+const presence = new Presence({
+	clientId: "1240257888514080830",
+});
+
+const enum Assets { // Other default assets can be found at index.d.ts
+	Logo = "https://cdn.discordapp.com/attachments/587336926533648409/1240591699034767380/mpp2.png?ex=66471e8e&is=6645cd0e&hm=b288e175fe293d96c41c4041773d370a231a52a4cdd818eaa88342731a6762f9&",
```
Comment (theusaf) [inline websites/M/MultiplayerPiano.net/presence.ts:6]: These Discord links are temporary and may not work. The link in the metadata should be fine.
```suggestion
	Logo = "https://www.virtualdrumming.com/icons-piano/icons-512.png",
```

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/M/MultiplayerPiano.net/presence.ts [hunk 9]
```diff
@@ -0,0 +1,55 @@
+const presence = new Presence({
+	clientId: "1240257888514080830",
+});
+
+const enum Assets { // Other default assets can be found at index.d.ts
```
Comment (DarkVillager) [inline websites/M/MultiplayerPiano.net/presence.ts:5]:
```suggestion
const enum Assets {
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Review by theusaf — CHANGES_REQUESTED** [review]
Make sure to resolve the lint check

**Review by theusaf — APPROVED** [review]

File: websites/M/MultiplayerPiano.net/metadata.json [hunk 10]
```diff
@@ -0,0 +1,36 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "224864947480297472",
+		"name": "antokek"
+	},
+	"service": "MultiplayerPiano.net",
```
Comment (Bas950) [inline websites/M/MultiplayerPiano.net/metadata.json:7]:
```suggestion
	"service": "Multiplayer Piano",
```
Same with folder name.

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by theusaf** [comment]
Closing for inactivity. You may open a new PR after resolving the issues mentioned in this one.

**Pull request closed without merging**